### ISSUE TITLE
CPU Remains High!

### ISSUE THREAD

**Issue by dcloud-wandl** [microsoft/vscode#232885]
Type: <b>Bug</b>

Menu->File->New Window
```
CMD: 
"D:\Microsoft VS Code\Code.exe" --type=utility --utility-sub-type=node.mojom.NodeService --lang=zh-CN --service-sandbox-type=none --dns-result-order=ipv4first --inspect-port=0 --video-capture-use-gpu-memory-buffer --user-data-dir="C:\Users\Dracula\AppData\Roaming\Code" --standard-schemes=vscode-webview,vscode-file --enable-sandbox --secure-schemes=vscode-webview,vscode-file --cors-schemes=vscode-webview,vscode-file --fetch-schemes=vscode-webview,vscode-file --service-worker-schemes=vscode-webview --code-cache-schemes=vscode-webview,vscode-file --field-trial-handle=4884,i,5163436895718007495,12889086174209836841,262144 --disable-features=CalculateNativeWinOcclusion,SpareRendererForSitePerProcess,WinDelaySpellcheckServiceInit,WinRetrieveSuggestionsOnlyOnDemand --variations-seed-version --mojo-platform-channel-handle=4880 /prefetch:14
```
VS Code version: Code 1.95.1 (65edc4939843c90c34d61f4ce11704f09d3e5cb6, 2024-10-31T05:14:54.222Z)
OS version: Windows_NT x64 10.0.26100
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-12700H (20 x 2688)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.80GB (4.93GB free)|
|Process Argv|--crash-reporter-id b4413cbc-fffd-4379-bc65-b982706bd448|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (51)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.5.0
mermaid-markdown-syntax-highlighting|bpr|1.6.7
vscode-better-align|cho|1.4.2
path-intellisense|chr|2.9.0
regex|chr|0.5.1
ts-worksheet|chw|0.8.27
md-table-editor|clo|0.2.0
vscode-eslint|dba|3.0.10
githistory|don|0.6.20
gitlens|eam|15.6.2
EditorConfig|Edi|0.16.4
prettier-vscode|esb|11.0.0
funnycode|ExB|1.0.4
code-runner|for|0.12.2
copilot|Git|1.243.0
copilot-chat|Git|0.22.1
selected-lines-count|gur|1.4.0
batch-rename-extension|Jan|0.0.6
vscode-peacock|joh|4.2.2
vscode-position|jtr|1.1.2
code-background|Kat|3.0.2
workspace-in-status-bar|koa|1.1.2
render-crlf|med|1.8.1
git-graph|mhu|1.30.0
milkdown|mir|0.0.15
vscode-filesize|mkx|3.2.1
vscode-language-pack-ja|MS-|1.95.2024103009
vscode-language-pack-zh-hans|MS-|1.95.2024103009
vscode-dotnet-runtime|ms-|2.2.2
powershell|ms-|2024.2.2
typescript-explorer|mxs|0.4.2
color-highlight|nau|2.8.0
live-code-metric|Omr|0.0.5
BongoCat|pix|0.0.1
material-icon-theme|PKi|5.12.0
quicktype|qui|23.0.170
tsserver-live-reload|rbu|1.0.1
vscode-sort-json|ric|1.20.0
markdown-preview-enhanced|shd|0.8.14
win-opacity|ska|1.1.7
grepc|stn|1.3.0
code-spell-checker|str|3.0.1
vscode-editor-group-minimizer|suh|1.3.2
errorlens|use|3.20.0
intellicode-api-usage-examples|Vis|0.2.9
vscodeintellicode|Vis|1.3.2
vscodeintellicode-completions|Vis|2.0.1
vscodeintellicode-insiders|Vis|1.1.10
code-translate|w88|1.0.20
pretty-ts-errors|Yoa|0.6.0
markdown-all-in-one|yzh|3.6.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
01bff139:31013167
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013

```

</details>

**Comment by defisym** [comment]
![Image](https://github.com/user-attachments/assets/984027e7-9f5f-4e39-b582-3a838b381950)
I also encountered this issue, VS Code eats up CPU, and the result of extension bisect turns out it's not an issue of any extension.
![Image](https://github.com/user-attachments/assets/c25795b8-48fd-4c7a-8e26-16ce97db2127)

**Comment by mjbvz** [comment]
Please follow [these instructions](https://github.com/Microsoft/vscode/wiki/Performance-Issues) to collect performance information. Share the output of `code-insiders --status`

**Comment by RZAlucard** [comment]
I encountered a similar problem. I used a profile and opened three windows:

Window A: empty.

Window B and C: used the same profile and had multiple plugins installed.

After I closed windows B and C, the CPU usage remained high.

Specific manifestation:

Operation steps: [Describe your operation steps in detail, such as: open profile, create window, install plugin, close window, etc.]
Expected result: After closing the window, the CPU usage should drop significantly.
Actual result: The CPU usage is still high.
![Image](https://github.com/user-attachments/assets/fa534ac9-fab9-443c-9fa6-05977554bccd)
